Doc/library/enum.rst

```diff
@@ -533,6 +538,9 @@ Data Types
          >>> ~Color.RED
          <Color.GREEN|BLUE: 6>
 
+      ..versionchanged:: 3.11 inversion no longer returns a flag with a negative
+      value
```

Comment:
```suggestion
      .. versionchanged:: 3.11
      
      Inversion of a flag now returns a positive value that is the
      union of all flags not in the given flag, rather than a negative value.
```

Fix the syntax, and be clearer and more explicit here